commoncrypto/Cargo.toml

```diff
@@ -11,11 +11,12 @@ license = "MIT"
 travis-ci = { repository = "malept/rust-commoncrypto" }
 
 [features]
+# default = ["ios-compat"]
+ios-compat = []
```

Comment:
Why does this need to be behind a feature? Shouldn't it just be gated on the OS?

Reply:
So tests can be run on macOS.

Reply:
I'm not convinced that's the right way to do it, but I'll do some research when I have some free time.